src/append_only_zks.rs

```diff
@@ -120,6 +120,37 @@ impl Azks {
             .await
     }
 
+    async fn preload_nodes_for_insertion<S: V2Storage + Sync + Send, H: Hasher>(
+        &self,
+        storage: &S,
+        _insertion_set: Vec<(NodeLabel, H::Digest)>,
+    ) -> Result<(), AkdError> {
+        let preload_depth = 10;
+        println!("-------------- preload depth = {} -------", preload_depth);
```

Comment:
```debug!``` or ```trace!``` so it'll log to the proper loggers. We should be avoiding println! since we might want to redirect output to another source (i.e. external log sink eventually)